diff --git a/src/Makefile.am b/src/Makefile.am
index 5b28579..6fa4d35 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -597,7 +597,7 @@ protoc_outputs =                                                  \
   google/protobuf/util/json_format_proto3.pb.cc                   \
   google/protobuf/util/json_format_proto3.pb.h
 
-BUILT_SOURCES = $(protoc_outputs)
+#BUILT_SOURCES = $(protoc_outputs)
 
 if USE_EXTERNAL_PROTOC
 
diff --git a/src/google/protobuf/stubs/common.h b/src/google/protobuf/stubs/common.h
index c362014..9efd2e1 100644
--- a/src/google/protobuf/stubs/common.h
+++ b/src/google/protobuf/stubs/common.h
@@ -1446,6 +1446,7 @@ LIBPROTOBUF_EXPORT bool IsStructurallyValidUTF8(const char* buf, int len);
 #include <byteswap.h>  // IWYU pragma: export
 
 #else
+#define GG_ULONGLONG(x) x##ULL
 
 static inline uint16 bswap_16(uint16 x) {
   return static_cast<uint16>(((x & 0xFF) << 8) | ((x & 0xFF00) >> 8));
